### PR TITLE
fix: add 24 hour ticket in recent purchase

### PR DESCRIPTION
**Pull request by Nisha1306**
Now when we have added 24 hour ticket as a new ticket type, it got missed from recent purchases. So have added it here.

### CODE REVIEW

File: src/screens/Ticketing/Tickets/RecentTickets/RecentTickets.tsx [hunk 1]
```diff
@@ -45,7 +45,8 @@ export const RecentTickets = () => {
         if (
           ticketType === 'single' ||
           ticketType === 'period' ||
-          ticketType === 'carnet'
+          ticketType === 'carnet' ||
+          ticketType === 'hour24'
```
Comment (gorandalum) [inline src/screens/Ticketing/Tickets/RecentTickets/RecentTickets.tsx:49]: Can't this be removed all together? 🤔  This should just be handled by the `productIsSellableInApp`.

**Review by gorandalum — APPROVED** [review]
Looks good, but I don't see why the ticket type check is necessary. Just remove it?

File: src/screens/Ticketing/Tickets/RecentTickets/RecentTickets.tsx (hunk 1 above)
Comment (gorandalum) [reply src/screens/Ticketing/Tickets/RecentTickets/RecentTickets.tsx:49]: Or let me think. Maybe it is necessary when adding new products 🤔  Just leave it 😅

Comment (Nisha1306) [reply src/screens/Ticketing/Tickets/RecentTickets/RecentTickets.tsx:49]: yeah...if we add new products other than these. so not touching it for now :)

**Pull request merged**